src/python/pants/backend/python/subsystems/pytest.py

```diff
@@ -44,6 +44,29 @@ def register_options(cls, register):
              help='Requirements string for the unittest2 library, which some python versions '
                   'may need.',
              removal_version="1.25.0.dev0", removal_hint="Use --pytest-plugins instead.")
+    register(
+      '--timeouts',
+      type=bool,
+      default=True,
+      help='Enable test target timeouts. If timeouts are enabled then test targets with a '
+          'timeout= parameter set on their target will time out after the given number of '
+          'seconds if not completed. If no timeout is set, then either the default timeout '
+          'is used or no timeout is configured.',
+    )
+    register(
+      '--timeout-default',
+      type=int,
+      advanced=True,
+      default=60,
```

Comment:
This doesn't have a default in V1 and shouldn't here.

However, do note that `--timeouts` is enabled by default. I think the reason for this is that it means by default, we won't ever timeout something. But, if someone wants timeouts, all they need to do is either set `--timeout-default` or add `timeout=` to a BUILD entry.